### DISCUSSION TITLE
Request to add "comma" to the #each helper

### DISCUSSION THREAD

**Discussion by jpeter06** [mockoon/mockoon#946] [Ideas]
Hello all,
I have been looking but **not found** a simple way of adding a "," separator between the #each elements.
Should be posible to add this option ( the same found in #repeat) to this helper?

Just an idea, I think is usefull feature.

thanks for all,

Juan Pedro.

**Comment by jpeter06** [comment]
Just found the solution `{{#unless  @index}}, {{/unless}}`

**Comment by jpeter06** [comment]
rectification: _**unless**_   doesn't do what I has expected :-|
At the end what I am doing is compare the index with the length of an array. But it sounds to complex.

**Reply by 255kb** [reply]
You can use `{{#unless @last}},{{/unless}}` (the boolean `@last` is available in `each`).
